### PR TITLE
Updated the styling

### PR DESCRIPTION
**Pull request by dk-raw**
**Made the images non-draggable and non-selectable. This improves the overall quality of the website**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -501,3 +501,18 @@ button.bg-primary:focus {
...
+  -moz-user-select: none !important;
+  -webkit-user-drag: none !important;
+  -webkit-user-select: none !important;
+  -ms-user-select: none !important;
+  -webkit-touch-callout: none !important;
+  -webkit-user-select: none !important;
+  -khtml-user-select: none !important;
+  -moz-user-select: none !important;
+  -ms-user-select: none !important;
+   user-select: none !important;
```
Comment (patricklodder) [inline assets/css/styles.css]: these seem to be duplicate entries of the same thing.

Comment (dk-raw) [reply assets/css/styles.css]: these are required for the different browsers

Comment (patricklodder) [reply assets/css/styles.css]:
```
$ sed -n 510p assets/css/styles.css |shasum 
5f2791809bee9b05ab16c9e243e07bb0ccbb3ddb  -
$ sed -n 513p assets/css/styles.css |shasum 
5f2791809bee9b05ab16c9e243e07bb0ccbb3ddb  -
```

the lines are exactly the same?

Comment (michilumin) [reply assets/css/styles.css]: > these are required for the different browsers

Yes, understood, but lines 510 and 513 are both "-webkit-user-select: none !important;"

Comment (dk-raw) [reply assets/css/styles.css]: Oh, I didn't saw that. let me fix it

Comment (patricklodder) [reply assets/css/styles.css]: the same for:
 
- 508 and 514
- 511 and 515

the rest is fine.

Comment (dk-raw) [reply assets/css/styles.css]: Fixed that one too. Sorry for the trouble, I guess I pasted it 2 times.

**Review by patricklodder — APPROVED** [review]
This is good now.

**Pull request merged**